src/DiIiS-NA/D3-GameServer/GSSystem/PlayerSystem/ConversationManager.cs

```diff
@@ -564,26 +562,27 @@ private void StopLine(bool interrupted)
 		/// <summary>
 		/// Starts readout and display of a certain conversation line
 		/// </summary>
-		/// <param name="LineIndex">index of the line withing the rootnodes</param>
-		private void PlayLine(int LineIndex)
+		/// <param name="lineIndex">index of the line withing the rootnodes</param>
+		private void PlayLine(int lineIndex)
 		{
-			if (asset.RootTreeNodes[LineIndex].ConvNodeType == 6)
+			if (asset.RootTreeNodes[lineIndex].ConvNodeType == 6)
 			{
 				currentLineNode = null;
 				return;
 			}
 
-			if (asset.RootTreeNodes[LineIndex].ConvNodeType == 4)
-				currentLineNode = asset.RootTreeNodes[LineIndex]
-					.ChildNodes[RandomHelper.Next(asset.RootTreeNodes[LineIndex].ChildNodes.Count)];
+			if (asset.RootTreeNodes[lineIndex].ConvNodeType == 4)
+				currentLineNode = asset.RootTreeNodes[lineIndex]
+					.ChildNodes[RandomHelper.Next(asset.RootTreeNodes[lineIndex].ChildNodes.Count)];
 			else
-				currentLineNode = asset.RootTreeNodes[LineIndex];
+				currentLineNode = asset.RootTreeNodes[lineIndex];
 
 			currentUniqueLineID = manager.GetNextUniqueLineID();
 
 			if (!GetSpeaker(currentLineNode.LineSpeaker).IsRevealedToPlayer(player))
 				GetSpeaker(currentLineNode.LineSpeaker).Reveal(player);
 
+			var duration = GetDuration();
```

Comment:
previously that prop was called thrice, so method semantically provokes us to cache the value